baseapp-content-feed/baseapp_content_feed/graphql/mutations.py

```
@@ -0,0 +1,57 @@
+import graphene
+
+from django import forms
+from graphene_django.types import ErrorType
+from graphene_django.forms.mutation import _set_errors_flag_to_context
+
+
+from baseapp_core.graphql import RelayMutation, login_required
+from baseapp_content_feed.models import SwappedContentPost as ContentPost
+from django.utils.translation import gettext_lazy as _
```

Ruff (0.7.0)

`django.utils.translation.gettext_lazy` imported but unused

Remove unused import: `django.utils.translation.gettext_lazy`

(F401)